code/modules/swarmers/swarmer_act.dm

```diff
@@ -164,9 +164,24 @@
 
 /turf/closed/wall/swarmer_act(mob/living/simple_animal/hostile/swarmer/actor)
 	var/isonshuttle = istype(loc, /area/shuttle)
+	var/pressure_greatest = 0
+	var/pressure_smallest = INFINITY 	//Freaking terrified to use INFINITY, man
+	for(var/t in RANGE_TURFS(2, src))	//Begin processing the delta pressure across the wall.
+		var/turf/open/turf_adjacent = t
+		if(!istype(turf_adjacent))
+			continue
+		if(turf_adjacent.air.return_pressure() > pressure_greatest)
+			pressure_greatest = turf_adjacent.air.return_pressure()
```

Comment:
```suggestion
			pressure_greatest = max(pressure_greatest, turf_adjacent.air.return_pressure())
```

Reply:
Hm, stylistically this is a good idea, however this results in a write operation eve if the max is unchanged. For optimization sake it might be best to leave as is.

Reply:
You aren't running a hot proc, you don't need to worry about that too much

Reply:
Aye aye!